### PR TITLE
Feat/export as csv

### PR DESCRIPTION
**Pull request by mgeerse**

### CODE REVIEW

File: functions/export-to-csv/2.0/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        new ExportToCsv({
+          fieldSeparator: delimiter,
+          quoteStrings: '"',
+          decimalSeparator: '.',
+          showLabels: true,
+          showTitle: false,
+          title: 'Diverted Documents Search Export ',
+          useTextFile: false,
+          useBom: true,
+          useKeysAsHeaders: true,
```
Comment (thomasvanophem) [inline functions/export-to-csv/2.0/index.js:23]: We could add all these fields to the options. That would mean that we, team actions, have to provide a checkbox option.

File: functions/export-to-csv/2.0/function.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
+{
+  "description": "Create a CSV from a collection of Records",
+  "label": "Export to CSV",
+  "category": "Utilities",
+  "icon": "CreateIcon",
```
Comment (thomasvanophem) [inline functions/export-to-csv/2.0/function.json:5]: I think we should use a different icon for the export function (team Actions will have to add the icons to the IDE and json schema first)

**Review by thomasvanophem — APPROVED** [review]

File: functions/export-to-csv/2.0/index.js (hunk 1 above)
Comment (mgeerse) [reply functions/export-to-csv/2.0/index.js:23]: @thomasvanophem I believe there's alreadya switch available (not a checkbox) but it does the same, I can add these into the `function.json` :)

Comment (mgeerse) [reply functions/export-to-csv/2.0/index.js:23]: @thomasvanophem I've added the options to `functions.json` but now some options like the titleName will still be shown even though the useTitle is disabled.

**Comment by joerigroot** [comment]
I added a loop functionality to this step. For a use case for Houthoff we needed to export more than 200 records. In the previous version of the step I could only select a collection with max 200 records. 

I also added an option to fill in property names (E.g. "id fullName emailAddress"). Previously all properties were exported.

**Pull request closed without merging**